bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -188,7 +190,7 @@ public void getURLInput(String url) throws IOException {
                             "Error communicating with Tibber API: " + e.getMessage());
                 }
             }
-            if (jsonResponse.contains("hourly")) {
+            if (jsonResponse.contains("hourly") && !jsonResponse.contains("[]") && !jsonResponse.contains("\"hourly\":null")) {
```

Comment:
same suggestion here.

Reply:
Good point, narrowed down to `"hourly":{"nodes":[]`